Sources/Jetworking/Client/Client.swift

```diff
@@ -357,6 +410,11 @@ public final class Client {
             )
         }
 
+        // Append additional header fields.
+        additionalHeaderFields.forEach { key, value in
```

Comment:
Should we do this after the interceptors have been executed? Maybe they overwrite a header field which should be set here? Don't know if the order in which the header fields are being added is of meaning.

Reply:
That is a point I'm not sure about the best implementation. Therefore I decided to merge the header fields. The order shouldn't be mandatory for http interfaces.

Reply:
There is a difference between `addValue` and `setValue` so we should in general discuss which we want. 

See following documentation:
```
/// If a value was previously set for the given header
    /// field, that value is replaced with the given value. Note that, in
    /// keeping with the HTTP RFC, HTTP header field names are
    /// case-insensitive.
    public mutating func setValue(_ value: String?, forHTTPHeaderField field: String)

    /// This method provides a way to add values to header
    /// fields incrementally. If a value was previously set for the given
    /// header field, the given value is appended to the previously-existing
    /// value. The appropriate field delimiter, a comma in the case of HTTP,
    /// is added by the implementation, and should not be added to the given
    /// value by the caller. Note that, in keeping with the HTTP RFC, HTTP
    /// header field names are case-insensitive.
    public mutating func addValue(_ value: String, forHTTPHeaderField field: String)
```

Reply:
Yes, I know. Since they are additional headers I would recommend to use `addValue` instead of `setValue` - Global Headers > Additional Headers.

Reply:
👍